content/en/docs/setup/independent/control-plane-flags.md

```diff
@@ -64,6 +67,13 @@ controllerManager:
     cluster-signing-key-file: /home/johndoe/keys/ca.key
     bind-address: 0.0.0.0
     deployment-controller-sync-period: 50
+    cloud-provider: gce
```

Comment:
These docs aim to be vendor-neutral, so how about
```suggestion
    cloud-provider: exampleprovider
```
?

I wonder if it makes sense to have two tabs, one that has the _extraVolumes_ and the cloud provider set and another that doesn't? I know that cluster operators definitely deploy with `kubeadm` onto bare metal, and that that's supported.

Reply:
yeah, cloud provider examples should not be included in this document.
see https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#kubeadm

Reply:
Agree not to include cloud provider here

Reply:
@neolit123 

By the way, I noticed that `--cloud-provider` is already deprecated when starting kubelet and kube-controller-manager, and it said the in-tree cloud providers will be moved to out-tree repos like https://github.com/kubernetes/cloud-provider-gcp

If I want to integrate the out-tree cloud provider code above, do you know how I can do this?

Reply:
unfortunately no. if you have GCP specific question you can try asking in the #sig-gcp slack channel.

Reply:
> unfortunately no. if you have GCP specific question you can try asking in the #sig-gcp slack channel.

I asked several days ago, but the channel is not active and there is no one replying me

Reply:
you can also try the #sig-cloud-provider channel.